src/proto/primitive.proto

```diff
@@ -184,3 +184,11 @@ message DirectControlPrimitive
     MotorControl motor_control = 1;
     PowerControl power_control = 2;
 }
+
+enum BallStealMode
+{
+    // The robot avoids stealing ball/getting possession
+    IGNORE = 0;
+    // The robot can steal the ball/try to get possession of the ball
+    STEAL = 1;
+}
```

Comment:
Note that the `ControlParams` of a FSM need to be one to one with that FSM's protobuf in `tactic.proto`. So you should add the new field here:
https://github.com/UBC-Thunderbots/Software/blob/3e4c785b47f91deff423c293da0ac4a9d9e7d987/src/proto/tactic.proto#L64-L72

This is so in our pytests we can specify all of the control params. With that change, you can probably move `BallStealMode`s definition to `tactic.proto` as well, similar to `CreaseDefenderAlignment`.

For the new proto to also be applied from the pytests, you would need to also update the `createTactic` function here:
https://github.com/UBC-Thunderbots/Software/blob/3e4c785b47f91deff423c293da0ac4a9d9e7d987/src/software/ai/hl/stp/tactic/tactic_factory.cpp#L70-L80

Reply:
Regarding your second point, is this still the case despite there existing a default value for the field or is it a readability improvement? `  tactic_proto.max_allowed_speed_mode()` also has a default value I believe.

Reply:
So if you don't pass in `tactic_proto.ball_steal_mode()` explicitely here, any pytest we would write (no matter what you set `ball_steal_mode`) will run with the default value of `updateControlParams`.